.travis.yml

```diff
@@ -26,6 +26,7 @@ before_script:
   - cp config/application.yml.example config/application.yml
   - cp certs/saml.crt.example certs/saml.crt
   - cp keys/saml.key.enc.example keys/saml.key.enc
+  - cp keys/pii_signing.key.enc.example keys/pii_signing.key.enc
```

Comment:
I think it's moot in this case because I have now reverted the affected line that this comment thread started on.

In general, I find refactoring a great thing. It does tend to make big PRs even bigger though, because the more files that get touched, the more there is to refactor. The tainting effect, like pulling on a thread in an old sweater.